src/openrct2/core/Compression.cpp

```diff
@@ -19,20 +19,21 @@
 
 #include <limits>
 #include <zlib.h>
+#include <zstd.h>
 
 namespace OpenRCT2::Compression
 {
-    constexpr size_t kZlibChunkSize = 128 * 1024;
-    constexpr size_t kZlibMaxChunkSize = static_cast<size_t>(std::numeric_limits<uInt>::max());
-    constexpr int kZlibWindowBits[] = { -15, 15, 15 + 16 };
+    static constexpr size_t kZlibChunkSize = 128 * 1024;
+    static constexpr size_t kZlibMaxChunkSize = static_cast<size_t>(std::numeric_limits<uInt>::max());
+    static constexpr int kZlibWindowBits[] = { -15, 15, 15 + 16 };
```

Comment:
What's the static here for?

Reply:
This was something I meant to do back when I first added these constants, but forgot. I figured I could do it here like with adding the `uLL` suffixes that @ZehMatt suggested. Short version: since these values are only declared in the source file, they should have internal linkage, meaning declared in an anonymous namespace or using the static keyword. It shouldn't affect how it works, but it's a change still worth making.